lib/api/1.1/southbound/profiles.js

```diff
@@ -47,16 +45,6 @@ function profilesRouterFactory (
                     });
                 }
             })
```

Comment:
I considered a short term workaround that was quite brittle because it relies on AMQP events only to track workflow start/finished state and can quickly become inaccurate if any of those are missed. My hope is that with the performance gains we are getting from the refactor, plus the logging updates and other improvements we made since the introduction of this, that we won't need it anymore.

If we start to see this become an issue again, or anticipate right now that it will, we should add this back in, but I think we should re-design it to be more robust in its tracking of state than just an AMQP event listener.